data/scopes/yealink-T21.ini

```diff
@@ -1,9 +1,18 @@
 [metadata]
 scopeType = "model"
 displayName = "Yealink T21P E2"
-version = 3
+version = 4
 
 [data]
+cap_contrast = ""
+contrast = 
+cap_brightness = ""
+brightness =
+cap_screensaver_file = ""
+screensaver_time = 600
```

Comment:
The `cap_screensaver_time_blacklist` variable is not set in yealink-*.ini files. However 004.php defines it. 

Please add it to all the Yealink scopes!